knowledge-graph/src/main/scala/io/renku/knowledgegraph/datasets/details/RequestedDataset.scala

```diff
@@ -54,16 +57,34 @@ object RequestedDataset {
     override def fold[A](idf: datasets.Identifier => A, saf: datasets.SameAs => A): A = saf(value)
   }
 
+  implicit def encoder(implicit renkuUrl: RenkuUrl): Codec[RequestedDataset] = Codec.from(
```

Comment:
since it's a full codec, perhaps we can name it also `codec` or similar?

Reply:
Sure